src/main/src/main/java/org/geoserver/ows/StylePublisher.java

```diff
@@ -0,0 +1,123 @@
+/* (c) 2014 Open Source Geospatial Foundation - all rights reserved
+ * (c) 2001 - 2013 OpenPlans
```

Comment:
As a new file it is probably (c) 2016, although I think we have recently updated our header policy.

* http://docs.geoserver.org/latest/en/developer/policies/committing.html